apps/mobile/src/app/(home)/settings/networks/index.tsx

```diff
@@ -7,22 +7,27 @@ import {
   DefaultNetworkConfigurations,
   WalletDefaultNetworkConfigurationIds,
 } from '@leather.io/models';
-import { GlobeIcon, PlaygroundFormsIcon, TestTubeIcon } from '@leather.io/ui/native';
+import {
+  Action,
+  GlobeIcon,
+  PlaceholderIcon,
+  PlaygroundFormsIcon,
+  TestTubeIcon,
+} from '@leather.io/ui/native';
 import { capitalize } from '@leather.io/utils';
 
 import SettingsScreenLayout from '../settings-screen.layout';
-import { NetworkCell } from './network-cell';
 
 function getNetworkIcon(network: DefaultNetworkConfigurations) {
```

Comment:
Maybe `getNetworkIcon` could instead be a component that returns a JSX version of the icon to help avoid the uppercase `Icon=`